extensions/project-descriptor.md

```diff
@@ -113,12 +154,12 @@ Any files that are excluded (either via `include` or `exclude`) MUST BE excluded
 
 If both `exclude` and `include` are defined, the build process MUST result in an error.
 
-### `[[build.buildpacks]]`
+#### `io.buildpacks.build.buildpacks` (optional)
```

Comment:
I don't think we need to abide by the `build.buildpacks` table name anymore, and it looks terrible (especially when we start tacking on `script` and other things.

I think we should change to this (or similar):

```
[[io.buildpacks.group]]
```

For example:

```
[[io.buildpacks.group]]
id = "io.buildpacks/java"
version = "1.0"

[[io.buildpacks.group]]
id = "io.buildpacks/nodejs"
version = "1.0"

[[io.buildpacks.group]]
id = "my/inline"

  [io.buildpacks.group.script]
  api = "0.4"
  inline = "echo 'hello!'"
```

Reply:
Alternative to `group` include `set`, `list`, `def`

Reply:
Another option is use `[[io.buildpacks]]` and buy another domain for the `build` table

Reply:
I like `group`, it matches `order.toml` and looks less silly :)

Reply:
I didn't read this comment closely enough the first time. I don't think it makes sense to remove `build` from `group` if we are keeping it for things like `env`. Instead maybe we should remove `build` entirely and reintroduce it at a lower layer if there is ambiguity in the future (e.g. `env.build` vs `env.launch`)

Reply:
I think it's probably ok to remove `build` altogether. I think it was originally there to protect against things like setting config for the exported image. But we can still have something like this in the future:

```
[[io.buildpacks.env]]
key = "FOO"
value = "build-time"

[[io.buildpacks.group]]
id = "io.buildpacks/java"
version = "1.0"

[[io.buildpacks.images]]
id = "my-app"

  [[io.buildpacks.images.env]]
  key = "FOO"
  value = "run-time"
```

Reply:
We decided for `env` since there is build now and maybe launch/runtime in the future to just have a `env.build` key.